kafl_fuzzer/manager/queue.py

```diff
@@ -92,8 +91,6 @@ def update_node_results(self, nid, results, new_payload):
             node.node_struct["info"]["trimmed"] = True
             node.set_score(self.scheduler.score_speed(node))
         if results.get("performance"):
-            oldperf = node.get_initial_performance()
```

Comment:
breaks the following out-commented print...either remove all or comment all?

Reply:
the print() could be converted to logger.debug()

Reply:
commented-out code is dead code,
I can convert it to logger.debug() for convenience.

Reply:
In my last update, I uncommented the print() dead code, and turned it into a  `logger.debug()` statement, keeping the 2 lines above of course